perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java

```diff
@@ -394,6 +394,9 @@ public void updateUserWithNullValueInLastName() throws Exception {
 	public void deleteUser() throws Exception {
 		System.out.println(CLASS_NAME + "deleteUser");
 
+		// Enable deletion of users
+		BeansUtils.getCoreConfig().setUserDeletionForced(true);
```

Comment:
please use try-finally to return CoreConfig properties to their original values (see the 'anonymizeUser()' test for reference)

Reply:
Tests rewrote to set the `userDeletionForced` property back to the original value.